### PR TITLE
Guilds remove invites when user rejects

### PR DESCRIPTION
**Pull request by TheHollidayInn**
This provides a reject url and removes invitations from groups. It should fix part of #5871

### CODE REVIEW

File: website/src/routes/apiv2.coffee [hunk 1]
```diff
@@ -478,6 +478,15 @@ module.exports = (swagger, v2) ->
       middleware: [auth.auth, i18n.getUserLanguage, groups.attachGroup]
       action: groups.join
 
+    "/groups/{gid}/reject":
+      spec:
+        method: 'POST'
+        description: 'Reject an invitation to a group'
+        parameters: [path('gid','Id of the group to join','string')]
```
Comment (crookedneighbor) [inline website/src/routes/apiv2.coffee]: This should be "Id of the group invitation to reject"

**Comment by TheHollidayInn** [comment]
Updated! Thanks for the revision.

**Comment by crookedneighbor** [comment]
Blocked by the api test pr.

**Comment by crookedneighbor** [comment]
No longer blocked! Just needs tests in the new structure.

**Comment by crookedneighbor** [comment]
We're accelerating our work on V3 of the API, as such, we're not going to accept anymore V2 PRs. Note that this is a desired feature, so we'll want it to be a part of v3.

**Comment by TheHollidayInn** [comment]
Does API three have routes for groups yet? I'm not seeing anything to add to.

**Pull request closed without merging**